ciscoasa.json

```diff
@@ -3,22 +3,20 @@
     "name": "Cisco ASA",
     "description": "This app supports containment actions in addition to investigative actions on a Cisco ASA device",
     "type": "firewall",
-    "publisher": "Splunk",
+    "publisher": "Splunk Community",
     "main_module": "ciscoasa_connector.py",
-    "app_version": "2.0.4",
+    "app_version": "3.0.0",
     "utctime_updated": "2022-01-07T20:56:17.000000Z",
     "package_name": "phantom_ciscoasa",
     "product_vendor": "Cisco Systems",
     "product_name": "Cisco ASA",
     "product_version_regex": ".*",
-    "min_phantom_version": "4.9.39220",
-    "latest_tested_versions": [
-        "On Prem, Version 7.6"
-    ],
+    "min_phantom_version": "5.1.0",
+    "fips_compliant": false,
     "python_version": "3",
     "logo": "logo_cisco.svg",
     "logo_dark": "logo_cisco_dark.svg",
-    "license": "Copyright (c) 2014-2021 Splunk Inc.",
+    "license": "Copyright (c) 2014-2022 Splunk Inc.",
```

Comment:
For reference, it should be sufficient to update the license year here. Additional job will be run to use the year here to update the license years of other parts.

Reply:
Thanks @paulnawat I will follow this process hence forth